mcda/utils/utils_for_main.py

```diff
@@ -323,9 +354,14 @@ def read_matrix(input_matrix_path: str) -> pd.DataFrame:
     :rtype: pd.DataFrame
     """
 
+    custom_input_path = os.environ.get('PROMCDA_INPUT_PATH')  # check if an environmental variable is set
+    input_directory_path = custom_input_path if custom_input_path else DEFAULT_INPUT_DIRECTORY_PATH
+
+    ensure_directory_exists(input_directory_path)
```

Comment:
- I believe that the user should be informed what path was selected: default or from env variable?
-  what if it the path doesn't exists? As of now I see you will create folders and will continue the execution. 
- I think here your code will also try to create folders if it doesn't exists. In my opinion it should not happen in case of `input_dir_path`. If the dir does not exists then inform the user that input dir is missing and exit the code. Otherwise, you might have to fail the execution at later point of time as there would not be any input files.

Reply:
What do you think about comment: 

> I believe that the user should be informed what path was selected: default or from env variable?

Reply:
What do you think about comment:

> I think here your code will also try to create folders if it doesn't exists. In my opinion it should not happen in case of input_dir_path. If the dir does not exists then inform the user that input dir is missing and exit the code. Otherwise, you might have to fail the execution at later point of time as there would not be any input files.

Reply:
> What do you think about comment:
> 
> > I believe the user should be informed what path was selected: default or from env variable?

That is the output on the terminal right now. The user is informed where the input matrix is read (first line) and where the output files are saved (third last line). Isn't it already what you wish to see?

<img width="1369" alt="Screenshot 2024-04-02 at 10 02 08" src="https://github.com/wetransform-os/ProMCDA/assets/26147427/8a087592-6b90-4fc7-a83f-decbc793c38e">

What modification do you exactly suggest?

Reply:
> What do you think about comment:
> 
> > I think here your code will also try to create folders if it doesn't exists. In my opinion it should not happen in case of input_dir_path. If the dir does not exists then inform the user that input dir is missing and exit the code. Otherwise, you might have to fail the execution at later point of time as there would not be any input files.

You are right. I removed the `ensure_directory_exists` check in the `read_matrix` function. If a non-existent directory path of the input matrix is now given, the code exits with this error:

<img width="1134" alt="Screenshot 2024-04-02 at 10 18 07" src="https://github.com/wetransform-os/ProMCDA/assets/26147427/8bb05b21-e3b0-4b00-ac65-960f1b0a8cba">

Reply:
Great, one more suggestion to catch the exception and log a meaningful message and then exit the code gracefully instead of these stack traces that are internal to python library. 
Something like this: https://stackoverflow.com/a/13425279